q2_diversity/_ordination.py

```diff
@@ -10,8 +10,19 @@
 import pandas as pd
 
 
-def pcoa(distance_matrix: skbio.DistanceMatrix) -> skbio.OrdinationResults:
-    return skbio.stats.ordination.pcoa(distance_matrix)
+def pcoa(distance_matrix: skbio.DistanceMatrix,
+         number_of_dimensions: int = None) -> skbio.OrdinationResults:
+    if number_of_dimensions == None:
+        # calculate full decomposition using eigh
+        return skbio.stats.ordination.pcoa(distance_matrix, method='eigh',
+                                           inplace=False)
+    else:
+        # calculate the decomposition only for the `number_of_dimensions`
+        # using fast heuristic eigendecomposition (fsvd)
+        return skbio.stats.ordination.pcoa(
+            distance_matrix, method='fsvd',
+            number_of_dimensions=number_of_dimensions,
+            inplace=False)
```

Comment:
If using FSVD I think the default should be `inplace=True` since the approximation also makes an implication about the size of the dataset and we know (from experience) that inplace=True can make or break your computation.